examples/message/SlackMessageExample.js

```diff
@@ -0,0 +1,94 @@
+const { Reshuffle } = require('reshuffle')
+const { SlackConnector, SlackEventType, SlackEvents } = require('reshuffle-slack-connector')
+const app = new Reshuffle()
+
+const main = async () => {
+  const slackConnector = new SlackConnector(app, {
+    token: '<your_slack_token_starting_with_xox>',
+    signingSecret: '<your_slack_signing_secret>',
+    port: '<slack_receiver_port>', // Default to 3000
+    endpoints: '<slack_receiver_endpoints>', // Default to '/'
+  })
+
+  // Events
```

Comment:
probably needs documentation on how to set these events up in Slack

Reply:
added